load_dqn_and_save_phi.py

```diff
@@ -0,0 +1,209 @@
+                    type=int,
+                    default=10000,
+                    metavar='STEPS',
+                    help='Number of training steps between evaluations')
+parser.add_argument('--evaluation-episodes',
+                    type=int,
+                    default=10,
+                    metavar='N',
+                    help='Number of evaluation episodes to average over')
+# TODO: Note that DeepMind's evaluation method is running the latest agent for 500K frames ever every 1M steps
+parser.add_argument('--evaluation-size',
+                    type=int,
+                    default=500,
+                    metavar='N',
+                    help='Number of transitions to use for validating Q')
+parser.add_argument('--render',
+                    action='store_true',
+                    help='Display screen (testing only)')
+parser.add_argument('--enable-cudnn',
+                    action='store_true',
+                    help='Enable cuDNN (faster but nondeterministic)')
+parser.add_argument('--checkpoint-interval',
+                    default=0,
+                    type=int,
+                    help='How often to checkpoint the model, \
+            defaults to 0 (never checkpoint)')
+parser.add_argument('--memory', help='Path to save/load the memory from')
+parser.add_argument('--phi-net-path',
+                    type=str,
+                    help='Path to save/load the phi network from')
+parser.add_argument('--disable-bzip-memory',
+                    action='store_true',
+                    help='Don\'t zip the memory file. Not recommended \
+            (zipping is a bit slower and much, much smaller)')
+parser.add_argument('--uuid',
+                    help="""UUID for the experient run.
+                    `env` uses environment name,
+                    `random` uses a random UUID
+                        (i.e. 2b25d648-ce34-4523-b409-35247d481e32),
+                    anything else is custom, example:
+                    PongNoFrameskip_ari_5 (uuid = env)
+                    PongNoFrameskip_64f9c4c9-bee5-44a6-9a61-d70267d9d623_ari_5 (uuid = random)
+                    PongNoFrameskip_lr_1e-5_ari_5 (uuid = lr_1e-5 (custom))
+                    """,
+                    default='env',
+                    type=str,
+                    required=False)
+
+# Setup
+args = parser.parse_args()
+
+if torch.cuda.is_available() and not args.disable_cuda:
+    args.device = torch.device('cuda')
+    torch.cuda.manual_seed(args.seed)
+    torch.backends.cudnn.enabled = args.enable_cudnn
+else:
+    args.device = torch.device('cpu')
+
+np.random.seed(args.seed)
+torch.manual_seed(args.seed)
+
+env, test_env, run_tag = get_env(args)
+dqn = Agent(args, env)
+
+model_filename = os.path.basename(args.model)
+model_dirname = os.path.dirname(args.model)
+
+phi_net_path = os.path.join(model_dirname, 'phi_'+model_filename)
+torch.save((dqn.online_net.phi, dqn.online_net.feature_size), phi_net_path)
```

Comment:
It was easy to forget to create the tuple here. Maybe a wrapper would save us in the future.

Reply:
Good point.